### PR TITLE
Unuse the gameplay clock in KeyCounter

### LINKED ISSUE #4993: Key counter lit animation slows down with the fail one
**Describe the bug:** The key counter `lit` animation seems to slow down with the fail animation. (Not a big issue, but that's probably not the correct design)

**Screenshots or videos showing encountered issue:** https://streamable.com/sd7o7

**osu!lazer version:** 2019.608.0

### PR DESCRIPTION
**Pull request by frenzibyte**
Using the gameplay clock causes the key counter to slow down due to the rate slowing down on failing state, unusing it should be better.

Closes #4993

### CODE REVIEW

File: osu.Game/Screens/Play/KeyCounter.cs [hunk 1]
```diff
@@ -75,9 +75,6 @@ protected KeyCounter(string name)
         [BackgroundDependencyLoader(true)]
         private void load(TextureStore textures, GameplayClock clock)
```
Comment (Altenhh) [inline osu.Game/Screens/Play/KeyCounter.cs]:
```suggestion
        private void load(TextureStore textures)
```

**Review by naoei — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
On a quick check, this PR in its current state breaks rewind logic (the counters no longer rewind when seeking backwards in a replay). Probably means we are short on tests.

**Review by peppy — CHANGES_REQUESTED** [review]
As mentioned, this regresses behaviour.

**Comment by iiSaLMaN** [comment]
I'm not sure if that is possible to fix...

**Comment by peppy** [comment]
If you don't think you have a solution in sight, closing this PR is a valid option. Breaking the replay support is far worse than potentially fixing #4993, which is but a visual glitch.

**Comment by iiSaLMaN** [comment]
I'll close it for now till i find a better fix for it, i have a lot of work to do for other PRs atm.

**Pull request closed without merging**